server/cluster/coordinator.go

```diff
@@ -155,6 +160,49 @@ func (c *coordinator) patrolRegions() {
 	}
 }
 
+// checkMissRegions check miss peer ,if miss region is more than maxMissPeerQueueSize,
+// it will return ture to avoid run other replica check
+func (c *coordinator) checkMissRegions() bool {
+	add := make([]*cache.Entry, 0)
+	remove := make([]uint64, 0)
+	missPeers := c.checkers.GetMissRegions()
+	regionListGauge.WithLabelValues("priority").Set(float64(len(missPeers)))
+	log.Debug("check miss regions", zap.Int("miss region count", len(missPeers)))
+	for _, entry := range missPeers {
+		id, ok := entry.Value.(uint64)
+		if !ok {
+			continue
+		}
+		region := c.cluster.GetRegion(id)
+		if region == nil {
+			// the region could be recent split, continue to wait.
+			remove = append(remove, id)
+			continue
+		}
+		if c.opController.GetOperator(id) != nil {
+			continue
+		}
+
+		ops := c.checkers.CheckRegion(region)
```

Comment:
We don't need to use CheckRegion since it's only for the miss region.

Reply:
it will not put into operator queue if operator kind is merge

Reply:
What I mean is the region could also enter the merge checker.

Reply:
it will discard if checkRegion retrun merge operators

Reply:
So why do we need to check merge here?

Reply:
check region may genrator merge operate, we hope pd should make up the miss replicas instance of merge